src/main/index.ts

```diff
@@ -27,14 +27,18 @@ process.env.VENCORD_USER_DATA_DIR = DATA_DIR;
 function init() {
     const { disableSmoothScroll, hardwareAcceleration } = Settings.store;
 
-    if (hardwareAcceleration === false) app.disableHardwareAcceleration();
+    if (hardwareAcceleration === false) {
+        app.disableHardwareAcceleration();
+    } else {
+        app.commandLine.appendSwitch("enable-features", "VaapiVideoDecodeLinuxGL,VaapiVideoEncoder,VaapiVideoDecoder");
```

Comment:
`enable-features` doesn't seen to be a [supported cli flag](https://www.electronjs.org/docs/latest/api/command-line-switches)

Reply:
https://chromium.googlesource.com/chromium/src/+/main/docs/configuration.md#Features

It's a chromium flag not a electron flag